core/storage/improvements/gae_models.py

```diff
@@ -120,7 +120,13 @@ def has_reference_to_user_id(cls, user_id):
 
     @staticmethod
     def get_deletion_policy():
-        """OK to delete task entries since they're just a historical record."""
+        """TaskEntryModel contains data to delete corresponding to
+        a user: resolver_id field.
+
+        It is okay to delete task entries since after they are resolved they
+        only act as a historical record and the removal does just removes
```

Comment:
```
It is okay to delete task entries since, after they are resolved, they only act as a historical record. The removal just removes the historical record.
```